src/MainArea/List/List.js

```diff
@@ -0,0 +1,23 @@
+import "./List.css";
+export const ListItem = ({ children }) => {
+  return (
+    <div>
+      <li>{children}</li>
+    </div>
+  );
+};
+export const List = ({ children, type }) => {
+  return (
+    <div>
+      {type === "ordered" && <ol>{children}</ol>}
+      {type === "unordered" && <ul>{children}</ul>}
+      {type === "no-bullet" && (
+        <div style={{ listStyleType: "none" }}>{children}</div>
```

Comment:
I would suggest avoiding inline styles. create a class instead.

Reply:
Yes, I'll correct that and will make this a habit not to use inline styles.